src/api/app/models/event.rb

```diff
@@ -1 +1,2 @@
-require_relative 'event/all'
+module Event
```

Comment:
I'm pretty sure this isn't necessary?

Reply:
If not `Event` is not defined. You'll get something like:

`LoadError: Unable to autoload constant Event, expected /obs/src/api/app/models/event.rb to define it`

Reply:
What line does that error come from?

Reply:
That's what you get if you try to access to `Event` from the console.

I created another branch exactly like this one, but deleting this two lines and that's the Travis result: https://travis-ci.org/Ana06/open-build-service/builds/312423246

As you can see it fails as Event definition is not where Rails expected it to be. So it can not be removed. 😉